committeeoversightapp/templates/detail.html

```diff
@@ -47,5 +47,13 @@
      <td><b>Witnesses</b></td>
      <td>{{witnesses}}</td>
    </tr>
+   <tr>
+     <td><b>Sources</b></td>
+     <td>
+       {% for source in hearing.sources.all %}
+        <a href={{source}}>{{source}}<br /></a>
```

Comment:
Would it be useful to add "target=_blank"?

Reply:
Yes! I've added this to the transcript links as well.